### PR TITLE
Re-add arguments check for v8 function bindings

### PR DESCRIPTION
**Pull request by edkilday**
The check for the correct number of arguments was removed (sorry) with the work to introduce properties.
This adds the check back in and, now, only checks for [in] arguments instead of both [in] and [out] arguments.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module Source
```
Comment (martinweismann) [inline go.mod:1]: what's that file doing here? :)

Comment (edkilday) [reply go.mod:1]: When I added a go extension to VSCode for a bit of help it complained until I added one of these.

Comment (edkilday) [reply go.mod:1]: I didn't think it would do any harm to submit and it may save someone some time setting up.

Comment (martinweismann) [reply go.mod:1]: OK

**Pull request merged**